Rubberduck.VBEEditor/ComManagement/ComMessagePumper.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Threading;
+using Rubberduck.VBEditor.ComManagement.VBERuntime;
+
+namespace Rubberduck.VBEditor.ComManagement
+{
+    public static class ComMessagePumper
+    {
+        private static SynchronizationContext UiContext { get; set; }
+        private static IVBERuntime Runtime { get; set; }
```

Comment:
I haven't seen this be set in the diff. I wonder whether it'd make sense to make this a Readonly and directly couple this to the `VBERuntimeAccessor`.

Reply:
Good catch. Fixed.

Reply:
Sorry, no. It's unfixed. :D What I didn't remember was that it's `static` and `readonly` can't be initialized in a non-static ctor.